modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/ChangeEmailViewModel.kt

```diff
@@ -80,9 +80,8 @@ class ChangeEmailViewModel
                         _changeEmailState.value = ChangeEmailState.Failure(errors, response.message)
                     }
                 }
-            }catch (e:Exception){
+            } catch (e: Exception) {
                 _changeEmailState.value = ChangeEmailState.Failure(mutableSetOf(ChangeEmailError.SERVER), e.message)
-
             }
         }
```

Comment:
Fixed a formatting issue that got auto-merged to `main` after CI run in https://github.com/Automattic/pocket-casts-android/pull/1514